src/Notes.h

```diff
@@ -11,6 +11,7 @@
 
 #include "acl/forward.h"
 #include "base/RefCount.h"
+#include "sbuf/SBuf.h"
```

Comment:
New header code does not store SBuf. Can we just do something like this instead?

```suggestion
#include "sbuf/forward.h"
```

Reply:
Yes, changing

Reply:
The promised change did not happen or was reverted. Unresolving. The question in the change request remains open.